app/components/Explorer/Accounts.jsx

```diff
@@ -177,14 +193,29 @@ class Accounts extends React.Component {
             },
             {
                 title: <Translate component="span" content="account.percent" />,
-                dataIndex: "accountPercentages",
-                key: "accountPercentages",
+                dataIndex: "accountBalance",
+                key: "accountBalancePercentage",
                 sorter: (a, b) => {
-                    a.accountPercentages = parseFloat(a.accountPercentages);
-                    b.accountPercentages = parseFloat(b.accountPercentages);
-                    return a.accountPercentages > b.accountPercentages
+                    let a_balance = 0;
```

Comment:
same logic as above? remove code duplication.

Also, are you certain that `accountPercentages` and `accountBalance` will always result in same sorting?

Reply:
Redundant code fixed. 

`accountPercentages` and `accountBalance` is always the same value, the balance object ID, as seen here https://github.com/bitshares/bitshares-ui/pull/2518/files/43ed20ff1751f6980eb17ce6b8bb6acb3da5cd90#diff-db5bfe202a0f1034f9c94ffab4f37ebaL245